internal/characters/klee/skill.go

```diff
@@ -69,19 +76,18 @@ func (c *char) Skill(p map[string]int) action.ActionInfo {
 		IsDeployable:       true,
 	}
 
-	//roughly 160 frames after mines are laid
 	for i := 0; i < minehits; i++ {
-		c.Core.QueueAttack(ai, combat.NewCircleHit(c.Core.Combat.Player(), 1, false, combat.TargettableEnemy), 0, skillStart+160, c.c2)
+		c.Core.QueueAttack(ai, combat.NewCircleHit(c.Core.Combat.Player(), 1, false, combat.TargettableEnemy), 0, mineHitmark, c.c2)
 	}
 
-	c.c1(skillStart + 30)
+	c.c1(bounceHitmarks[0])
 
-	c.SetCD(action.ActionSkill, 1200)
+	c.SetCDWithDelay(action.ActionSkill, 1200, 33)
```

Comment:
This is bugged when used with `.ready.klee.skill`. Not sure what to do here to fix it, other than to propose `c.SetCD(action.ActionSkill, 1233)`.